WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -0,0 +1,159 @@
+import UIKit
+
+final class SiteInformationWizardContent: UIViewController {
+    fileprivate enum Rows: Int, CaseIterable {
+        case title = 0
+        case tagline = 1
+
+        static func count() -> Int {
+            return allCases.count
+        }
+
+        func matches(_ row: Int) -> Bool {
+            return row == self.rawValue
+        }
+    }
+
+    private let completion: (SiteInformation) -> Void
+
+    @IBOutlet weak var table: UITableView!
+    @IBOutlet weak var nextStep: UIButton!
+
+    private lazy var headerData: SiteCreationHeaderData = {
+        let title = NSLocalizedString("Basic information", comment: "Create site, step 3. Select basic information. Title")
+        let subtitle = NSLocalizedString("Tell us more about the site you are creating.", comment: "Create site, step 3. Select basic information. Subtitle")
+
+        return SiteCreationHeaderData(title: title, subtitle: subtitle)
+    }()
+
+    init(completion: @escaping (SiteInformation) -> Void) {
+        self.completion = completion
+        super.init(nibName: String(describing: type(of: self)), bundle: nil)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        applyTitle()
+        setupBackground()
+        setupTable()
+        setupNextButton()
+    }
+
+    private func applyTitle() {
+        title = NSLocalizedString("2 of 3", comment: "Site creation. Step 2. Screen title")
+    }
+
+    private func setupBackground() {
+        view.backgroundColor = WPStyleGuide.greyLighten30()
+    }
+
+    private func setupTable() {
+        setupTableBackground()
+        registerCell()
+        setupHeader()
+
+        table.dataSource = self
+        table.delegate = self
+    }
+
+    private func setupTableBackground() {
+        table.backgroundColor = WPStyleGuide.greyLighten30()
+    }
+    private func registerCell() {
+        table.register(
+            InlineEditableNameValueCell.defaultNib,
```

Comment:
TIL we have an `InlineEditableNameValueCell`! :)

Reply:
One curiosity of that type - its text field is of type `LoginTextField`. 

It appears that this is _already_ used outside of authentication contexts, but should it be?

Reply:
Sorry, but I am not sure I understand the question.

Reply:
The subview of `InlineEditableNameValueCell` is `LoginTextField`, which suggests that this cell is intended for use with authentication fields. If so, should we be using it for a purpose other than authentication?